app/src/main/java/com/space/quizapp/presentation/ui/start/StartFragment.kt

```diff
@@ -8,25 +8,25 @@ import android.view.View
 import android.view.ViewGroup
 import com.space.quizapp.R
 
-class MainFragment : Fragment() {
+class StartFragment : Fragment() {
```

Comment:
სტარტს აუთენტიფიკაცია ჯობია, მგონი შენი იდეა იყო

Reply:
changed in a later pull request